node/primitives/bridge/src/lib.rs

```diff
@@ -211,18 +206,6 @@ pub fn derive_account_from_pangoro_id(id: SourceAccount<AccountId>) -> AccountId
 /// - check that the sender has paid enough funds for both message delivery and dispatch.
 #[derive(RuntimeDebug)]
 pub struct DarwiniaFromThisChainMessageVerifier<B, R>(PhantomData<(B, R)>);
```

Comment:
```suggestion
pub struct FromThisChainMessageVerifier<B, R>(PhantomData<(B, R)>);
```

Same issue as I mentioned in another PR.

Darwinia indicates the Darwinia chain, and I also have ideas to use another name as prefixes of pallet developed by us.